File: app/src/main/java/com/sdk/billinglibrary/CrossedTextView.java
package com.sdk.billinglibrary;

import android.content.Context;
import android.graphics.Canvas;
import android.graphics.Color;
import android.graphics.Paint;
import android.util.AttributeSet;

public class CrossedTextView extends androidx.appcompat.widget.AppCompatTextView {

    private Paint paint;

    public CrossedTextView(Context context, AttributeSet attrs) {
        super(context, attrs);
        init();
    }
    public CrossedTextView(Context context) {
        super(context);
        init();
    }

    public CrossedTextView(Context context, AttributeSet attrs, int defStyle) {
        super(context, attrs, defStyle);
        init();
    }

    private void init() {
        paint = new Paint();
        paint.setColor(Color.RED);
        paint.setStrokeWidth(getResources().getDisplayMetrics().density * 1);
    }

    @Override
    protected void onDraw(Canvas canvas) {
        super.onDraw(canvas);
        canvas.drawLine(0, getHeight()/2, getWidth(), getHeight()/2, paint);
    }

}

File: app/src/main/java/com/sdk/billinglibrary/CrossedTextView.kt
package com.sdk.billinglibrary

import android.content.Context
import android.graphics.Canvas
import android.graphics.Color
import android.graphics.Paint
import android.util.AttributeSet
import androidx.appcompat.widget.AppCompatTextView

class CrossedTextView : AppCompatTextView {

    private var paint: Paint? = null

    constructor(context: Context) : super(context) {
        init()
    }

    constructor(context: Context, attrs: AttributeSet?) : super(context, attrs) {
        init()
    }

    constructor(context: Context, attrs: AttributeSet?, defStyle: Int) : super(context, attrs, defStyle) {
        init()
    }

    private fun init() {
        paint = Paint()
        paint!!.color = Color.RED
        paint!!.strokeWidth = resources.displayMetrics.density * 1
    }

    override fun onDraw(canvas: Canvas) {
        super.onDraw(canvas)
        canvas.drawLine(0f,
            (height / 2).toFloat(),
            width.toFloat(),
            (height / 2).toFloat(),
            paint!!)
    }
}
